### PR TITLE
Update UI-Component for standalone

### PR DESCRIPTION
**Pull request by ivy-edp**

### CODE REVIEW

File: .ivy/raise-deps.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 set -e
 
 sed -i -E "s/(\"@axonivy[^\"]*\"): \"[^\"]*\"/\1: \"~${1/SNAPSHOT/next}\"/" packages/*/package.json
+sed -i -E "s/(\"@axonivy[^\"]*\"): \"[^\"]*\"/\1: \"~${1/SNAPSHOT/next}\"/" integration/*/package.json
```
Comment (ivy-lmu) [inline .ivy/raise-deps.sh:5]: same here integrations

**Review by ivy-lmu — APPROVED** [review]

**Comment by ivy-edp** [comment]
> btw, `npm dedupe` could help you clean up the lock file

nice thanks, didnt know this command but after appling it, it seems to work now 👍

**Comment by ivy-edp** [comment]
@ivy-lli fyi

**Comment by ivy-lli** [comment]
thx 👍

**Pull request merged**